pages/index.js

```diff
@@ -1,14 +1,18 @@
 import styled from "styled-components";
+import { useState } from "react";
 
 import ShopHeader from "@/components/ShopHeader";
 import Gallery from "@/components/Gallery";
 
 export default function HomePage() {
+  const [cartTotal, setCartTotal] = useState(0);
+
```

Comment:
```suggestion
function handleCartTotal(counter) {
      setCartTotal(cartTotal + counter);
    
  }
```

Reply:
I yes I totally forgot the controlled states.
Thanks a lot 👍

Reply:
- [x] ADD: handle function
- [x] FIX: props pass-through